scale-tests/configs/2018-11-15-mwst10.env

```diff
@@ -0,0 +1,145 @@
+# Depends on:
+# - TEST_NAME
+# - TEST_S3_BUCKET
+# - TEST_S3_FOLDER
+
+# Cluster configuration ########################################################
+
+CLUSTER_URL="https://mwst9.scaletesting.mesosphe.re"
+SECURITY="strict"
+
+# Test configuration ###########################################################
+
+SHOULD_INSTALL_INFRASTRUCTURE=true
+SHOULD_INSTALL_NON_GPU_DISPATCHERS=true
+SHOULD_INSTALL_GPU_DISPATCHERS=true
+SHOULD_RUN_FAILING_STREAMING_JOBS=false
+SHOULD_RUN_FINITE_STREAMING_JOBS=true
+SHOULD_RUN_INFINITE_STREAMING_JOBS=true
+SHOULD_RUN_BATCH_JOBS=true
+SHOULD_RUN_GPU_BATCH_JOBS=true
+SHOULD_UNINSTALL_INFRASTRUCTURE_AT_THE_END=false
+
+# Infrastructure configuration #################################################
+
+SERVICE_NAMES_PREFIX="${TEST_NAME}/"
+INFRASTRUCTURE_OUTPUT_FILE="infrastructure.json"
+KAFKA_ZOOKEEPER_CONFIG='scale-tests/configs/kafka-zookeeper-options.json'
+KAFKA_CLUSTER_COUNT=1
+KAFKA_CONFIG='scale-tests/configs/kafka-options.json'
+CASSANDRA_CLUSTER_COUNT=1
+CASSANDRA_CONFIG='scale-tests/configs/cassandra-options.json'
+
+# Non-GPU dispatchers configuration ############################################
+
+NON_GPU_NUM_DISPATCHERS=50
+NON_GPU_DISPATCHERS_OUTPUT_FILE="non-gpu-dispatchers.out"
+NON_GPU_DISPATCHERS_JSON_OUTPUT_FILE="${NON_GPU_DISPATCHERS_OUTPUT_FILE}-dispatchers.json" # NOTE: this name is built internally by the deploy-dispatchers.py script.
+NON_GPU_QUOTA_DRIVERS_CPUS=20
+NON_GPU_QUOTA_DRIVERS_MEM=50000
+NON_GPU_QUOTA_EXECUTORS_CPUS=25
+NON_GPU_QUOTA_EXECUTORS_MEM=40000
+# Total quota:
+# - driver:
+#   - 1000 cpus
+#   - 2500000 mem
+# - executor:
+#   - 1250 cpus
+#   - 2000000 mem
+
+# GPU dispatchers configuration ################################################
+
+GPU_NUM_DISPATCHERS=5
+GPU_DISPATCHERS_OUTPUT_FILE="gpu-dispatchers.out"
+GPU_DISPATCHERS_JSON_OUTPUT_FILE="${GPU_DISPATCHERS_OUTPUT_FILE}-dispatchers.json" # NOTE: this name is built internally by the deploy-dispatchers.py script.
+GPU_QUOTA_DRIVERS_CPUS=16
+GPU_QUOTA_DRIVERS_MEM=200000
+GPU_QUOTA_DRIVERS_GPUS=
+GPU_QUOTA_EXECUTORS_CPUS=
+GPU_QUOTA_EXECUTORS_MEM=
+GPU_QUOTA_EXECUTORS_GPUS=
+# NOTE: to test Core team’s hypothesis of GPU quota negatively impacting spark
+# launch rates, we remove the GPU quota entirely from all executor roles.
+GPU_REMOVE_EXECUTORS_ROLES_QUOTAS=true
+# Total quota:
+# - driver:
+#   - 80 cpus
+#   - 1000000 mem
+#   - 0 gpus
+# - executor:
+#   - 0 cpus
+#   - 0 mem
+#   - 0 gpus
+
+# Common streaming jobs configuration ##########################################
+
+TEST_ASSEMBLY_JAR_URL='http://infinity-artifacts.s3.amazonaws.com/scale-tests/dcos-spark-scala-tests-assembly-20180612-5fa9420.jar'
```

Comment:
I don't think anything's changed in the source code since June but if yes then this will change depending on how far we get w/ updating all the latest SHAs, yeah?